src/segmantic/i2i/translate.py

```diff
@@ -44,7 +56,7 @@ def load_pix2pix_generator(
     return gen, device
 
 
-def load_cyclegan_generator(model_file_path: str, gpu_ids: list = []):
+def load_cyclegan_generator(model_file_path: Path, gpu_ids: Sequence[int] = []):
```

Comment:
Any special reason to annotate the last param as a ``Sequence`` instead of ``List`` (provided that afterwards you cast asa ``list``?
Look at this table of [``collections.abc``](https://docs.python.org/3/library/collections.abc.html#collections-abstract-base-classes) to determine which is the most appropriate

Reply:
true. thanks